src/api/block/controllers/block.ts

```diff
@@ -17,32 +17,51 @@ export default factories.createCoreController(
     async findByVuid(ctx) {
       try {
         const { vuid } = ctx.params
-        const { populate, locale } = ctx.query
+        const {
+          populate,
+          locale,
+          fallbackToDefaultLocale = 'false',
+        } = ctx.query
 
-        const entries: Block[] = await strapi.entityService.findMany(
-          'api::block.block',
-          {
+        let entries: Block[] = []
+
+        const requestedLocaleEntries: Block[] =
```

Comment:
Maybe `entries` can be initialized here instead? According to the API it should always return a list, even if empty.

Reply:
Hehe yeah, I thought about that too. I thought the code got more self-explanatory by doing this, but I now agree that it's not worth the detour